static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -173,6 +181,11 @@
     background: $paid-article-subheader;
     border-top-color: $paid-article-brand;
 
+    &:hover,
+    &:focus {
+        background: darken($paid-article-subheader, 7%);
```

Comment:
yeah I agree, although with colours it's very hard to do, unless you switch to the lab colour space—adding 5% to an RGB colour can produce very different optical results based on the amount of white in the colour.